core/src/main/java/com/radixdlt/ledger/InvalidCommitRequestException.java

```diff
@@ -71,7 +71,14 @@
  * consistent (e.g. non-parseable or non-committable transactions, ledger proof mismatches).
  */
 public class InvalidCommitRequestException extends RuntimeException {
+
+  private final InvalidCommitRequestError error;
+
   public InvalidCommitRequestException(InvalidCommitRequestError error) {
-    super(error.toString());
+    this.error = error;
```

Comment:
No magic is that powerful...
I probably deleted this accidentally (and never noticed, since we don't log this exception - just translate its `error` to a higher-level exception). Lemme bring it back, it won't hurt 👍